src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/BindingStep/help.html

```diff
@@ -3,20 +3,29 @@
     Each binding will define an environment variable active within the scope of the step.
     You can then use them directly from any other steps that expect environment variables to be set:
 </p>
-<pre><code>
-node {
-  withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'mylogin',
-                    usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD']]) {
+<pre><code>node {
+  withCredentials([usernameColonPassword(credentialsId: 'mylogin', variable: 'USERPASS')]) {
```

Comment:
Of course only works with sufficiently new Pipeline plugins, but I will assume that is most people going forward. _Snippet Generator_ will always produce syntax compatible with what you actually have.